### PR TITLE
spell out DCS

### PR DESCRIPTION
**Pull request by strouja**
Since this was an introduction page I thought it would be a good idea to spell out DCS the 1st time it was used on this page.  If you don't agree I won't be offended.  Take care, James

### CODE REVIEW

File: docs/README.rst [hunk 1]
```diff
@@ -79,7 +79,7 @@ Patroni can be installed with pip:
 where dependencies can be either empty, or consist of one or more of the following:
 
 etcd or etcd3
-    `python-etcd` module in order to use Etcd as DCS
+    `python-etcd` module in order to use Etcd as Distributed Donfiguration Store (DCS)
```
Comment (bradnicholson) [inline docs/README.rst:82]: s/Donfiguration/Configuration/

**Comment by strouja** [comment]
I clicked resolved for the conversation but I'm not sure if that was needed or made sense

**Comment by CyberDem0n** [comment]
@strouja there is exactly a spelling typo in your change

**Review by strouja — COMMENTED** [review]
fixed typo in my changed C to a D

**Comment by FxKu** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Comment by CyberDem0n** [comment]
Merged, thank you @strouja!

**Pull request merged**